docs/MachineConfigController.md

```diff
@@ -45,8 +45,16 @@ type MachineConfigPoolSpec struct {
     // This includes generating new desiredMachineConfig and update of machines.
     Paused bool `json:"paused"`
 
-    // MaxUnavailable specifies the percentage or constant number of machines that can be updating at any given time.
-    // default is 1.
+    // maxUnavailable defines either an integer number or percentage
+    // of nodes in the corresponding pool that can go Unavailable during
+    // an upgrade. This includes nodes Unavailable for any reason, including
+    // user initiated cordons, failing nodes, etc. The default value is 1.
```

Comment:
does it make sense to make `user initiated cordonos` to `user/other operator initiated cordons`

Reply:
I just added the user initiated part as an example, since I've seen users ask about it. I can add more examples if you think that makes sense

Reply:
fine as it is.